flow-typed/npm/jest_v27.x.x.js

```diff
@@ -1,6 +1,35 @@
 // flow-typed signature: 5ddcf688200e3506308fdcfa78ca48d9
 // flow-typed version: 644a595e77/jest_v27.x.x/flow_>=v0.134.x
 
+/**
```

Comment:
Should we do like we do in webapp and the `_CONTAINS_KA_CUSTOMIZATIONS` bit to the filename also?